homeassistant/components/nissan_leaf/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Battery Charge and Range Support for the Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_BATTERY, DATA_CHARGING, DATA_LEAF, DATA_RANGE_AC, DATA_RANGE_AC_OFF,
+    LeafEntity)
+from homeassistant.const import DEVICE_CLASS_BATTERY
+from homeassistant.helpers.icon import icon_for_battery_level
+from homeassistant.util.distance import LENGTH_KILOMETERS, LENGTH_MILES
+from homeassistant.util.unit_system import IMPERIAL_SYSTEM, METRIC_SYSTEM
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+ICON_RANGE = 'mdi:speedometer'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Sensors setup."""
+    _LOGGER.debug("setup_platform nissan_leaf sensors, discovery_info=%s",
```

Comment:
Remove this.

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/119b2a3b20db6ea4cf05fda29921930c5e27bc86

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/119b2a3b20db6ea4cf05fda29921930c5e27bc86